views/js/qtiCreator/widgets/static/helpers/itemScrollingMethods.js

```diff
@@ -22,59 +22,100 @@ define([
 ], function ($, typeCaster) {
     'use strict';
 
+    const wrapperTextCls = 'custom-text-box';
+    const wrapperIncludeCls = 'custom-include-box';
+    const wrapperFocusCls = 'key-navigation-focusable';
+    const newUIclass = 'tao-overflow-y';
+    const options = [{
+        value: '100',
+        name: 'Full height',
+        class: 'tao-full-height'
+    }, {
+        value: '75',
+        name: '3/4 of height',
+        class: 'tao-three-quarters-height'
+    }, {
+        value: '66.6666',
+        name: '2/3 of height',
+        class: 'tao-two-thirds-height'
+    }, {
+        value: '50',
+        name: 'Half height',
+        class: 'tao-half-height'
+    }, {
+        value: '33.3333',
+        name: '1/3 of height',
+        class: 'tao-third-height'
+    }, {
+        value: '25',
+        name: '1/4 of height',
+        class: 'tao-quarter-height'
+    }];
+
     return {
         options: function () {
-            return [{
-                value: '100',
-                name: 'Full height'
-            }, {
-                value: '75',
-                name: '3/4 of height'
-            }, {
-                value: '66.6666',
-                name: '2/3 of height'
-            }, {
-                value: '50',
-                name: 'Half height'
-            }, {
-                value: '33.3333',
-                name: '1/3 of height'
-            }, {
-                value: '25',
-                name: '1/4 of height'
-            }];
+            return options;
         },
-        isScrolling: function (wrapper) {
-            return typeCaster.strToBool(wrapper.length > 0 ? wrapper.attr('data-scrolling') : 'false');
+        isScrolling: function ($wrapper) {
+            return typeCaster.strToBool($wrapper.length > 0 ? $wrapper.attr('data-scrolling') : 'false');
         },
-        selectedHeight: function (wrapper) {
-            return wrapper.attr('data-scrolling-height');
+        selectedHeight: function ($wrapper) {
+            return $wrapper.attr('data-scrolling-height');
         },
         initSelect: function ($form, isScroll, height) {
             isScroll ? $form.find('.scrollingSelect').show() : $form.find('.scrollingSelect').hide();
             height && $form.find('.scrollingSelect select').val(height).change();
         },
         wrapContent: function (widget, value, wrapType) {
             const $form = widget.$form;
-            let $wrap = wrapType === 'inner' ? widget.$container
+            let $wrapper = wrapType === 'inner' ? widget.$container
                 .find('[data-html-editable]')
-                .children('.custom-text-box') : widget.$container.parent('.custom-text-box');
+                .children(`.${wrapperTextCls}`) : widget.$container.parent(`.${wrapperIncludeCls}`);
```

Comment:
I believe `parent` should be replaced with `closest`
```suggestion
                .children(`.${wrapperTextCls}`) : widget.$container.closest(`.${wrapperIncludeCls}`);
```